### ISSUE TITLE
Does ConditionalAccumulator have any other reduction_type?

### ISSUE THREAD

**Issue by bitianchi** [tensorflow/tensorflow#25876]
![image](https://user-images.githubusercontent.com/29396595/52999122-ee88f000-345f-11e9-94bf-72550d222b1d.png)

When I use SyncReplicasOptimizer, I want to sum the gradients but not to mean them. The motivation to do this is, I want to train a big model with big batch_size other than mini batch_size. So SyncReplicasOptimizer has any other reduction_types? (For example sum, max...)

**Comment by ymodak** [comment]
You can try other loss reduction types such as https://www.tensorflow.org/api_docs/python/tf/losses/Reduction
Thanks!
